boards/xtensa/qemu_xtensa/qemu_xtensa_defconfig

```diff
@@ -1,6 +1,7 @@
 CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_IPM_CONSOLE_STACK_SIZE=2048
 CONFIG_XTENSA=y
-CONFIG_SOC_SAMPLE_CONTROLLER=y
 CONFIG_BOARD_QEMU_XTENSA=y
 CONFIG_CONSOLE=y
+CONFIG_SOC_XTENSA_SAMPLE_CONTROLLER=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=10000000
```

Comment:
There's a HAL function to get this information in runtime.  If these values are out of sync, would this become a problem?